.github/workflows/libjs-test262.yml

```diff
@@ -13,6 +13,11 @@ jobs:
     concurrency: libjs-test262
 
     steps:
+      - name: Cleanup
+        run: |
+          echo "Cleaning up previous run"
+          sudo rm -rf "${{ github.workspace }}/*"
```

Comment:
Doesn't it run as root already? I commonly see apt install in workflows without sudo...

Reply:
oh right, it does, will remove!